src/_pytest/tmpdir.py

```diff
@@ -45,8 +45,21 @@ def from_config(cls, config) -> "TempPathFactory":
             given_basetemp=config.option.basetemp, trace=config.trace.get("tmpdir")
         )
 
+    def _check_basename(self, basename: str):
```

Comment:
I think this conveys better the method's purpose:

```suggestion
    def _ensure_relative_to_basetemp(self, basename: str):
```